weather_api/views.py

```diff
@@ -0,0 +1,48 @@
+import os
+import requests
+
+from datetime import datetime
+from django.contrib import messages
+from django.shortcuts import render
+
+from .forms import CityForm
+
+
+def index(request):
```

Comment:
to nie powinno się nazywać index, tylko nazwa powinna odpowiadać temu za co odpowiada ten endpoint